docs/README.md

```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+-->
+
+# Documentation
+
+To build the documentation:
+
+#. Install [Doxygen][doxygen].
+#. From this directory, run `doxygen`.
```

Comment:
This doesn't seem to be valid Markdown:
<img width="453" alt="image" src="https://github.com/user-attachments/assets/f5deac6e-ea4e-47c8-bb0e-e5f5e39a24d1" />

```suggestion
1. Install [Doxygen][doxygen].
1. From this directory, run `doxygen`.
```